app/models/audit/AuditTaskInteractionTable.scala

```diff
@@ -261,6 +261,218 @@ object AuditTaskInteractionTable {
     ).first
   }
 
+  /**
+    * Calculate combined time spent auditing across, we take the important events from the audit_task_interaction table,
+    * get the difference between each consecutive timestamp, filter out the timestamp diffs that are greater than five
+    * minutes, and then sum those time diffs.
+    */
+  def countTotalAuditing(): Float = db.withSession { implicit session =>
+    Q.queryNA[Float](
+      s"""SELECT CAST(
+         |  extract(second from SUM(diff)) / 60 +
+         |  extract(minute from SUM(diff)) +
+         |  extract(hour from SUM(diff)) * 60 AS decimal(10,2)) / 60.0 AS hours_audited
+         |FROM (
+         |  SELECT (timestamp - LAG(timestamp, 1) OVER(ORDER BY timestamp)) AS diff
+         |  FROM audit_task_interaction_small
+         |) "time_diffs"
+         |WHERE diff < '00:05:00.000' AND diff > '00:00:00.000';""".stripMargin
+    ).first
+  }
+
+  /**
+    * Calculate the combined time spent auditing today.
+    */
+  def countTodayAuditing(): Float = db.withSession { implicit session =>
+    Q.queryNA[Float](
+      s"""SELECT CAST(
+         |  extract(second from SUM(diff)) / 60 +
+         |  extract(minute from SUM(diff)) +
+         |  extract(hour from SUM(diff)) * 60 AS decimal(10,2)) / 60.0 AS hours_audited
+         |FROM (
+         |  SELECT (timestamp - LAG(timestamp, 1) OVER(ORDER BY timestamp)) AS diff
+         |  FROM audit_task_interaction_small
+         |  WHERE (timestamp AT TIME ZONE 'US/Pacific') > (NOW() AT TIME ZONE 'US/Pacific') - interval '1 day'
```

Comment:
For consistency with how we've been doing this elsewhere, instead of doing
```
(timestamp AT TIME ZONE 'US/Pacific') > (NOW() AT TIME ZONE 'US/Pacific') - interval '1 day'
```

let's do
```
(timestamp AT TIME ZONE 'US/Pacific')::date = (NOW() AT TIME ZONE 'US/Pacific')::date
```

So that it is actually for just the data "today" instead of "past 24 hours).